wordpress-mobile/WordPress-Android
# [Reader Improvements] Increase post actions padding

Context: p1697239330196399-slack-C05N140C8H5

To test:
- Install JP and sign-in;
- Open any tab of reader;
- Tap the post actions (e.g. `"Reblog"`,  `"Like"`): the tappable area size should be bigger than what we previously had.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
--

PR submission checklist:


UI Changes testing checklist:
